src/test/java/seedu/exercise/logic/commands/CustomRemoveCommandTest.java

```diff
@@ -15,35 +27,123 @@
 import seedu.exercise.model.ModelManager;
 import seedu.exercise.model.ReadOnlyResourceBook;
 import seedu.exercise.model.UserPrefs;
+import seedu.exercise.model.property.CustomProperty;
+import seedu.exercise.model.property.PropertyBook;
+import seedu.exercise.model.resource.Exercise;
+import seedu.exercise.testutil.builder.ExerciseBookBuilder;
+import seedu.exercise.testutil.builder.ExerciseBuilder;
 
 public class CustomRemoveCommandTest {
 
-    private Model model = new ModelManager(getTypicalExerciseBook(), new ReadOnlyResourceBook<>(),
-        new ReadOnlyResourceBook<>(), new ReadOnlyResourceBook<>(), new UserPrefs(), getDefaultPropertyBook());
+
+    private Map<String, String> customPropertiesMap = Map.of("Remark", "Text");
+    private Exercise firstExercise = new ExerciseBuilder().withCustomProperties(customPropertiesMap).build();
+    private Exercise secondExercise = new ExerciseBuilder(WALK).withCustomProperties(customPropertiesMap).build();
+    private ReadOnlyResourceBook<Exercise> exerciseBook = new ExerciseBookBuilder().withExercise(firstExercise)
+        .withExercise(secondExercise).build();
+    private Model model = new ModelManager(exerciseBook, new ReadOnlyResourceBook<>(),
+        new ReadOnlyResourceBook<>(), new ReadOnlyResourceBook<>(), new UserPrefs());
+    private PropertyBook propertyBook = PropertyBook.getInstance();
 
     @BeforeEach
     public void reset() {
```

Comment:
standard naming for @BeforeEach annotated methods is setUp()

Reply:
Noted and changed.